oak/proto/application.proto

```diff
@@ -71,3 +72,11 @@ message StorageProxyConfiguration {
   // The address of the external storage provider.
   string address = 1;
 }
+
+// GrpcClientConfiguration describes the configuration of a gRPC client
+// pseudo-Node (which is provided by the Oak Runtime), connected to a specific
+// external (non-Oak) gRPC service.
+message GrpcClientConfiguration {
+  // The endpoint address of the external gRPC service.
+  string address = 1;
```

Comment:
Maybe `server_address`? Also mention that this is `address:port`, and whether `http(s)` need to be specified too (and which one is implied, if any, if not). We should also somehow figure out / clarify what's the story around TLS (i.e. how are TLS certs validated?)

Reply:
"Endpoint address" is somewhat deliberately vague &ndash; the allowed syntax is going to be a bit environment specific, probably boiling down to something like "whatever `grpc.Dial()` supports in your deployment environment".